src/main/java/com/juniorstart/juniorstart/model/audit/UserStatus.java

```diff
@@ -0,0 +1,8 @@
+package com.juniorstart.juniorstart.model.audit;
+
+public enum UserStatus {
+    Looking_for_a_job,
+    Hired,
+    Freelancer,
+    Open
+}
```

Comment:
```suggestion
public enum USER_STATUS {
    LOOKING_FOR_A_JOB,
    HIRED,
    FREELANCER,
    OPEN
}
```

Reply:
Chciałem z małej bo idzie to do frontu jako lista statusów.

Reply:
Front sobie poradzi z zamianą tego na odpowiednie statusy widoczne dla użytkownika. Musimy robić tak back, żeby był zgodny z panującymi zasadami pisania czystego kodu.